CppCoreGuidelines.md

```diff
@@ -17696,7 +17696,45 @@ Type erasure incurs an extra level of indirection by hiding type information beh
 
 ##### Example
 
-    ???
+    #include <memory>
+
+    class Object
+    {
+    public:
+        template <typename T>
+        Object(std::unique_ptr<T> obj)
```

Comment:
any particular reason you are working with a `unique_ptr<T>` instead of just a `T`?

Reply:
I wanted to avoid copy operator. But I can use std::forward. I change my commit.